index.bs

```diff
@@ -7178,108 +8285,212 @@ partial dictionary MLOpSupportLimits {
   </thead>
   <tr>
     <td>{{input}}</td>
-    <td>{{MLOperandDataType/"float32"}}, {{MLOperandDataType/"float16"}}</td>
-    <td>4</td>
+    <td>[=/any data type|any=]</td>
+    <td>1 to [=/any rank|N=]</td>
+  </tr>
+  <tr>
+    <td>{{indices}}</td>
+    <td>{{MLOperandDataType/"int32"}}, {{MLOperandDataType/"uint32"}}, {{MLOperandDataType/"int64"}}</td>
+    <td>{{input}}'s [=MLOperand/rank=]</td>
+  </tr>
+  <tr>
+    <td>{{updates}}</td>
+    <td>[=/same type as|same as=] {{input}}</td>
+    <td>{{input}}'s [=MLOperand/rank=] and {{indices}}'s [=MLOperand/shape=]</td>
   </tr>
   <tr>
     <td>*output*</td>
     <td>[=/same type as|same as=] {{input}}</td>
-    <td>4</td>
+    <td>{{input}}'s [=MLOperand/rank=]</td>
   </tr>
 </table>
 
-{{MLOpSupportLimits}} has the following member for {{MLGraphBuilder/resample2d()}}:
+{{MLScatterSupportLimits}} has the following members:
+<dl dfn-type=dict-member dfn-for=MLScatterSupportLimits>
+    : <dfn>input</dfn>
+    :: {{MLSupportLimits}} for input operand.
+    : <dfn>indices</dfn>
+    :: {{MLSupportLimits}} for indices operand.
+    : <dfn>updates</dfn>
+    :: {{MLSupportLimits}} for updates operand.
+    : <dfn>output</dfn>
+    :: {{MLSupportLimits}} for output operand.
+</dl>
+
+{{MLOpSupportLimits}} has the following members for {{MLGraphBuilder/scatterElements()}}:
 <dl dfn-type=dict-member dfn-for=MLOpSupportLimits>
-    : <dfn>resample2d</dfn>
-    :: Support limits for operator {{MLGraphBuilder/resample2d()}}.
+    : <dfn>scatterElements</dfn>
+    :: Support limits for operator {{MLGraphBuilder/scatterElements()}}.
 </dl>
 
+<div class="note">
+  The {{MLGraphBuilder/scatterElements(input, indices, updates, options)/indices}} parameter to {{MLGraphBuilder/scatterElements()}} can not be clamped to the allowed range when the graph is built because the inputs are not known until execution. Implementations can introduce {{MLGraphBuilder/clamp()}} in the compiled graph if the required clamping behavior is not provided by the underlying platform. Similarly, if the underlying platform does not support negative indices, the implementation can introduce operations in the compiled graph to transform a negative index from the end of the dimension into a positive index.
```

Comment:
Address RFC2119 warning:
```suggestion
  The {{MLGraphBuilder/scatterElements(input, indices, updates, options)/indices}} parameter to {{MLGraphBuilder/scatterElements()}} can not be clamped to the allowed range when the graph is built because the inputs are not known until execution. Implementations can introduce {{MLGraphBuilder/clamp()}} in the compiled graph if the specified clamping behavior is not provided by the underlying platform. Similarly, if the underlying platform does not support negative indices, the implementation can introduce operations in the compiled graph to transform a negative index from the end of the dimension into a positive index.
```

Reply:
Arigatou.